knimin/handlers/ag_new_barcode.py

```diff
@@ -68,17 +69,19 @@ def post(self):
                    % num_barcodes)
 
         elif action == "assign":
-            projects = self.get_arguments('projects')
+            projects = [url_unescape(p).encode('utf-8')
```

Comment:
I am happy that you managed to pass all test, so maybe we don't want to add more changes, but shouldn't tornado automatically do those escapings?